mod/msearch.php

```diff
@@ -39,4 +41,4 @@ function msearch_post(&$a) {
 
 	killme();
 
-}
```

Comment:
Don't miss these, some Editors (under Linux) may crash if a last new-line character is missing.